docs/Developers/Newcomers_Guide.rst

```diff
@@ -158,7 +158,7 @@ fork repository.
    will work.
 
    ::
```

Comment:
After `::` enter a two newline and then four spaces with respect to `::` put `$ git checkout -b <branchname>`.
It will look something like the following

    ::

        $ git checkout -b <branchname>